server/src/main/java/org/elasticsearch/common/logging/ECSJsonLayout.java

```diff
@@ -64,13 +64,12 @@ public EcsLayout build() {
         }
 
         private KeyValuePair[] additionalFields() {
-            return new KeyValuePair[]{
-                new KeyValuePair("type",type),
-                new KeyValuePair("cluster.uuid","%cluster_id"),
-                new KeyValuePair("node.id","%node_id"),
-                new KeyValuePair("node.name","%ESnode_name"),
-                new KeyValuePair("cluster.name","${sys:es.logs.cluster_name}"),
-            };
+            return new KeyValuePair[] {
+                new KeyValuePair("event.dataset", type),
```

Comment:
also, do you think it would worthy to change the property used in log4j2.properties file ? from `type_name` to `dataset`?

Reply:
I've changed `type` and `type_name` to `dataset` throughout. Do you know whether there's anything breaking about doing that?